schemas/2015-11-01/Microsoft.Network.json

```diff
@@ -0,0 +1,231 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-11-01/Microsoft.Network.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Network",
+  "description": "Microsoft Network Resource Types",
+  "resourceDefinitions": {
+    "trafficManagerProfiles": {
+      "type": "object",
+      "description" : "An Azure Traffic Manager profile",
+      "properties": {
+        "apiVersion": {
+          "enum": [
+            "2015-11-01"
+          ]
+        },
+        "type": {
+          "enum": [
+            "Microsoft.Network/trafficManagerProfiles"
+          ]
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "profileStatus": {
+              "anyOf": [
```

Comment:
Agreed, switching back to one of, cheers

From: Wenjun Wu [mailto:notifications@github.com]
Sent: Tuesday 8 December 2015 21:20
To: Azure/azure-resource-manager-schemas <azure-resource-manager-schemas@noreply.github.com>
Cc: Vlad Cananau <Vlad.Cananau@microsoft.com>
Subject: Re: [azure-resource-manager-schemas] Schema for Traffic Manager Profile resource (#78)


In schemas/2015-11-01/Microsoft.Network.json<https://github.com/Azure/azure-resource-manager-schemas/pull/78#discussion_r47015135>:

> +      "properties": {

> +        "apiVersion": {

> +          "enum": [

> +            "2015-11-01"

> +          ]

> +        },

> +        "type": {

> +          "enum": [

> +            "Microsoft.Network/trafficManagerProfiles"

> +          ]

> +        },

> +        "properties": {

> +          "type": "object",

> +          "properties": {

> +            "profileStatus": {

> +              "anyOf": [

Vlad, please still use "oneOf", we need exact one match here, instead of "at least one". The error may be an issue of our test tool, I will work with @daschult<https://github.com/daschult> on it.

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-resource-manager-schemas/pull/78/files#r47015135>.